geomstats/geometry/product_manifold.py

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
 """Product of manifolds."""
+=======
+"""Define a product of manifolds."""
```

Comment:
Linked to @opeltre comment: we need to agree on a convention for the docstrings at top of the modules.